cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -95,6 +103,16 @@ function ManagementCredentials(props: Partial<StepProps>) {
     };
 
     const onSubmit: SubmitHandler<FormInputs> = (data) => {
+        const valueList = getValues();
+        for (const i in valueList) {
+            console.log(i);
```

Comment:
Sorry just noticed this. Since these are account credentials, we should probably not log them to the browser console.

Reply:
Yes sir. My bad. This is supposed to be printed for test. Removing them. 😄